doorway-ui-components/src/headers/SiteHeader.tsx

```diff
@@ -63,7 +63,7 @@ const SiteHeader = (props: SiteHeaderProps) => {
     const logoOffset = document.getElementById("site-header-logo")?.offsetLeft
     const linksOffset = document.getElementById("site-header-links")?.offsetLeft
     if (linksOffset === undefined || logoOffset === undefined) return
-    return linksOffset === 0
+    return linksOffset === 0 || linksOffset === logoOffset
```

Comment:
I vaguely remember removing this in the past, could you explain what case it covers?

Reply:
The issue here was that languages that overflow had a screen size in between mobile and tablet where it wasn't popping down to the next level and was overflowing the logo. It's not 100% visually clean though, mostly because of your callout in the screenshot above where the gray background doesn't always extend all the way.

Reply:
Will poke Em about getting a more thorough design here, I think the popping down piece was a dev add at some point